### ISSUE TITLE
[Audit] Purpose of "find out more" link unclear

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#218]
The purpose of the "Find out more" link on the [homepage](https://govuk-brand-guidelines.netlify.app/) is unclear out of context.
That fails WCAG 2.4.4 Link Purpose (In Context).

**Comment by calvin-lau-sig7** [comment]
We can make the minor edits to these links – the homepage call-to-action might be tricky depending on how meaningful it needs to be.
